### PR TITLE
handling missing chart data

### PR DESCRIPTION
**Pull request by emre2038**
## Description
The assumption of every `children` key of subindicators would have `count` property caused some charts to not work. Added some controls to fix that

## Related Issue
https://trello.com/c/QqmElVl5/495-ensure-that-the-fe-assumes-a-0-for-count-where-data-is-missing-in-order-to-not-break-other-aspects-of-the-site

## How to test it locally
* Navigate to a geography that has missing count property, for example `/#geo:EC` on `sifar-wazi.openup.org.za` (this could be done by hardcoding `hostname` parameter in `index.js`).
* Check if the charts like `Marital Status` are working.
* Note that some charts like `Older Adults Population` are still not working because _ALL_ the count properties are missing so they are all assigned 0.

## Screenshots
image
image

### Updated
* Updated `chart.js` to add necessary checks

## Checklist

- [ ]  🚀 is the code ready to be merged and go live?
- [ ]  🛠 does it work (build) locally

### Pull Request

- [ ]  📰 good title
- [ ]  📝good description
- [ ]  🔖 issue linked
- [ ]  📖 changelog filled out

### Commits

- [ ]  commits are clean
- [ ]  commit messages are clean

### Code Quality

- [ ]  🚧 no commented out code
- [ ]  🖨 no unnecessary logging
- [ ]  🎱 no magic numbers
- [ ]  ⚙️ ran jslint
- [ ]  🧰 ran codeclimate locally

### Testing

- [ ]  ✅ added (appropriate) unit tests
- [ ]  💢 edge cases in tests were considered
- [ ]  ✅ ran tests locally & are passing

### CODE REVIEW

**Comment by emre2038** [comment]
[Ensure that the FE assumes a 0 for count where data is missing in order to not break other aspects of the site](https://trello.com/c/QqmElVl5/495-ensure-that-the-fe-assumes-a-0-for-count-where-data-is-missing-in-order-to-not-break-other-aspects-of-the-site)

File: src/401.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <!--  This site was created in Webflow. http://www.webflow.com  -->
-<!--  Last Published: Tue Sep 22 2020 11:38:29 GMT+0000 (Coordinated Universal Time)  -->
+<!--  Last Published: Thu Oct 01 2020 07:27:52 GMT+0000 (Coordinated Universal Time)  -->
```
Comment (milafrerichs) [inline src/401.html:3]: please don't commit code that has not really changed. thanks 🙏

File: src/js/profile/chart.js [hunk 2]
```diff
@@ -91,7 +91,7 @@ export class Chart extends Observable {
     getValuesFromSubindicators = () => {
         let arr = [];
         for (const [label, subindicator] of Object.entries(this.subindicators)) {
-            let count = subindicator.count;
+            let count = isNaN(subindicator.count) ? 0 : subindicator.count;
```
Comment (milafrerichs) [inline src/js/profile/chart.js:94]: this is not the right place to make this change.
subindicator is a class `SubIndicator` and they have a `MISSING_VALUE` constant. 
For now let's change that from `N/A` to `0` and we should make a pin and refactor this, so we can change the MISSING_VALUE for a subindicator.

**Review by milafrerichs — CHANGES_REQUESTED** [review]
thanks for the PR but I think this is the wrong place do add another check.
please change the `MISSING_VALUE` constant for `SubIndicator`

File: src/js/profile/chart.js (hunk 2 above)
Comment (emre2038) [reply src/js/profile/chart.js]: done

**Review by adieyal — APPROVED** [review]
